scss/_utilities_sticky-footer.scss

```diff
@@ -1,17 +0,0 @@
-@mixin vf-u-sticky-footer {
```

Comment:
Did we decide we want to remove a sticky footer utility as well?

Reply:
The sticky footer util only related to the footer component. This reverts sticky footers to being a site-specific customisation.

Reply:
Oh ok, in that case it all looks good EXCEPT there's still a mention of `$sticky-footer` in the spacing settings file

Reply:
Ok, removed